default-menu-auto-hide.sh

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (C) 2022  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Red Hat Author(s): Jiri Konecny <jkonecny@redhat.com>
+
+# We have this feature on Fedora, however, only for Workstation which is not
```

Comment:
I guess you can add a test for `inst.profile=fedora-workstation` if you want to cover Fedora as well.

Reply:
I was thinking about that but I'm afraid of doing that because Workstation is a lot different from what we have in the boot.iso so the whole test would be fragile. Do you think it's fine to go this way @poncovka even when the Workstation is expecting Live and Live payload?

Reply:
Not Vendy but - I don't like the idea, if I read it right the tested combination would be "unsupported".

Reply:
In that case, we could just include a configuration snippet with `menu_auto_hide = True` on Fedora images. We already do that for some tests. See `anaconda-conf.sh`.